docs/en/ingest-management/getting-started.asciidoc

```diff
@@ -65,45 +53,43 @@ include::elastic-agent/elastic-agent.asciidoc[tag=agent-install-intro]
 
 To send logs and metrics to the {stack}:
 
-. On the **Agents** tab in {fleet}, click **Add agent**, and look at the
-deployment instructions under **Enroll in {fleet}**.
+. On the **Agents** tab in {fleet}, click **Add agent**.
 
-. As instructed, download and extract the {agent} to your host. To do this
-quickly from the command line, run:
+. Under **Enroll in {fleet}**, follow the in-product installation steps.
++
+[role="screenshot"]
+image::images/add-agent.png[]
 +
---
-include::elastic-agent/install-elastic-agent.asciidoc[tag=install-elastic-agent]
-
 See the https://www.elastic.co/downloads/elastic-agent[download page] for other
 installation options.
---
 
-. Back in {fleet}, under **Choose an agent policy**, notice that the default
-policy is selected. The default policy includes a system integration for
-collecting logs and metrics from the host system. Use the default policy to get
-started quickly.
+*Notes:*
 
-. Under **Enroll and start the {agent}**, copy the install command if it's
-available for your platform, or make a note of the {kib} URL and enrollment
-token that was generated by {fleet}.
-+
-[role="screenshot"]
-image::images/kibana-fleet-enroll.png[{fleet} showing agent enrollment page]
+//QUESTION: The build of cloud I'm using doesn't show a default policy. Is that
+//a bug, or expected?
```

Comment:
I think this question is more for Jason than me, but where are you seeing this? I'm not able to replicate on ESS staging.

Reply:
I saw it on cloud staging yesterday, but it sounds like there was a problem. I'll just remove this comment because I think we should test the quick starts again when everything is working for real.